modules/network-observability-resources-table.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+// network_observability/configuring_operator.adoc
+
+:_content-type: REFERENCE
+[id="network-observability-resources-table{context}"]
+= Resource considerations
+The following table outlines examples of resource considerations for clusters with certain workload sizes. 
+
+[IMPORTANT]
+====
+The examples outlined in the table demonstrate scenarios that are tailored to specific workloads. Consider each example only as a baseline from which adjustments can be made to accommodate your workload needs.
+====
+
+.Resource recommendations
+[options="header"]
+|============================================================================================================================
+|                                     | Extra small (10 Nodes) | Small (25 nodes)  | Medium (65 Nodes) ^[2]^ | Large (120 nodes) ^[2]^
+| *Worker Node vCPU and Memory*       | 4 vCPUs\| 16GiB Mem ^[1]^ | 16 vCPUs\| 64GiB Mem ^[1]^ | 16 vCPUs\| 64GiB Mem  ^[1]^  |16 vCPUs\| 64GiB Mem ^[1]^ 
+| *LokiStack Size*                    | 1x.extra-small         | 1x.small          | 1x.small           | 1x.medium
+| *NetObserv Controller Memory Limit* | 400Mi (default)        | 400Mi (default)   | 400Mi (default)    | 800Mi                
+| *eBPF Sampling Rate*                | 50 (default)           | 50 (default)      | 50 (default)       | 50 (default)
+| *eBPF Memory Limit*                 | 800Mi (default)        | 800Mi (default)   | 2000Mi             | 800Mi (default) 
+| *FLP Mem Limit*                     | 800Mi (default)        | 800Mi (default)   | 800Mi (default)    | 800Mi (default)         
+| *FLP Kafka Partitions*              | N/A                    | 48                | 48                 | 48            
+| *Kafka Consumer Replicas*           | N/A                    | 24                | 24                 | 24
+| *Kafka Brokers*                     | N/A                    | 3 (default)       | 3 (default)        | 3 (default)
```

Comment:
Nit: Usually headers (both the top actual header and the manual left-side one) should be in sentence case. Idk if the following would be fully accurate, but potentially something like:

```suggestion
|                                     | Extra small (10 nodes) | Small (25 nodes)  | Medium (65 nodes) ^[2]^ | Large (120 nodes) ^[2]^
| *Worker node vCPU and memory*       | 4 vCPUs\| 16GiB mem ^[1]^ | 16 vCPUs\| 64GiB mem ^[1]^ | 16 vCPUs\| 64GiB mem  ^[1]^  |16 vCPUs\| 64GiB mem ^[1]^ 
| *LokiStack size*                    | `1x.extra-small`         | `1x.small`          | `1x.small`           | `1x.medium`
| *NetObserv controller memory limit* | 400Mi (default)        | 400Mi (default)   | 400Mi (default)    | 800Mi                
| *eBPF sampling rate*                | 50 (default)           | 50 (default)      | 50 (default)       | 50 (default)
| *eBPF memory limit*                 | 800Mi (default)        | 800Mi (default)   | 2000Mi             | 800Mi (default) 
| *FLP memory limit*                     | 800Mi (default)        | 800Mi (default)   | 800Mi (default)    | 800Mi (default)         
| *FLP Kafka partitions*              | N/A                    | 48                | 48                 | 48            
| *Kafka consumer replicas*           | N/A                    | 24                | 24                 | 24
| *Kafka brokers*                     | N/A                    | 3 (default)       | 3 (default)        | 3 (default)
```

Also I noticed in the footnotes after this table that the sizes were marked up (e.g.,  `M6i.12xlarge`), so I showed marking them up in the above as well, in case that's valid.

Reply:
Good suggestions thank you! Addressed.